proto/zetachain/zetacore/observer/events.proto

```diff
@@ -15,6 +16,13 @@ message EventBallotCreated {
   string ballot_type = 5;
 }
 
+message EventBallotDeleted {
```

Comment:
Do we even need to have this event? In this case, each vote will have `EventBallotCreated` and `EventBallotDeleted`

Reply:
Same answer as this : https://github.com/zeta-chain/node/pull/2863#discussion_r1894339848